tests/phpunit/tests/test-wpml-elementor-integration-factory.php

```diff
@@ -26,6 +26,7 @@ public function it_creates() {
 			                     'WPML_Elementor_Adjust_Global_Widget_ID_Factory',
 			                     'WPML_PB_Elementor_Handle_Custom_Fields_Factory',
 			                     'WPML_Elementor_Media_Hooks_Factory',
+			                     'WPML_Elementor_WooCommerce_Hooks_Factory',
```

Comment:
@kagg-design What is this for?

Reply:
@brucepearson it is the fix of previous bug. Test failed without it even before my MR.